src/assets/BDDWithFlutter/BDDWithFlutter.md

```diff
@@ -175,7 +174,7 @@ Remember: The step `Then I expect the "counter" to be "10"` is a built-in-step.
 1. connect your phone or start the emulator
 2. run `dart test_driver/app_test.dart`
 
-after a while, you should see output like:
+After a while, you should see output like:
```

Comment:
> The sentence in line 190 is not complete (Github doesn't allow me to comment on the corresponding line, but since it is kind of the second part of this sentence I add this comment here)
> 
> Because line 177 ends with a double point I think the sentence in line 190 needs to be reworded to become a proper full sentence. Maybe add something like this to line 190: "Additionally, you should see the app working on the phone screen:"

`and the app working on the phone screen.` connects with the previous line, which seems okay if we read the blog.

Reply:
Grammatically there shouldn't be a double point in this case but `...` or no punctuation mark at the end of line 177 and line 190 should start with `... and the app` if this is considered to be the end of this sentence.

Reply:
will be addressed in https://github.com/JankariTech/blog/issues/150